config.toml

```diff
@@ -15,7 +15,7 @@ logo = "images/assets/logos/logo.png"
 description = "We see planet-wide possibilities."
 author = "Rotational Labs, LLC"
 googleAnalytics = "G-2FKX6CWJHW" # Google Analytics ID
-g_recaptcha_key="xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx" # Google reCAPTCHA key
+# g_recaptcha_key="6Ld5O3kiAAAAAJU0z0h81X1RxEMHyoROe6KWe_vk" # Google reCAPTCHA key
```

Comment:
Can this be added to a .env file? I'm not sure what the best practice is here with Hugo so want to double check.